src/GraphQL/ApiKeyAuthenticator.php

```diff
@@ -16,7 +16,7 @@
  */
 class ApiKeyAuthenticator implements AuthenticatorInterface
 {
-    public function authenticate(HTTPRequest $request)
```

Comment:
will this ever return null - maybe it could be more strongly typed? Member gets checked if it exists and is of type Member so it seems little chance it would be null after that?

Reply:
ah fair matching the interface is probably best 👍